packages/react-components/src/organisms/TeamGroupCard.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import css from '@emotion/css';
+import { ListGroupResponse } from '@asap-hub/model';
+import {
+  Card,
+  Paragraph,
+  Headline3,
+  Button,
+  Divider,
+  Headline4,
+} from '../atoms';
+import {
+  perRem,
+  mobileScreen,
+  smallDesktopScreen,
+  vminLinearCalcClamped,
+} from '../pixels';
+import { teamIcon } from '../icons';
+import { charcoal } from '../colors';
+
+const LESS_GROUP_LIMIT = 2;
+
+const teamsStyles = css({
+  color: charcoal.rgb,
+  display: 'flex',
+  alignItems: 'center',
+  paddingTop: `${12 / perRem}em`,
+  paddingBottom: vminLinearCalcClamped(
+    mobileScreen,
+    18,
+    smallDesktopScreen,
+    24,
+    'px',
+  ),
+});
+
+const iconStyles = css({
+  display: 'inline-block',
+  width: `${24 / perRem}em`,
+  height: `${24 / perRem}em`,
+  paddingRight: `${15 / perRem}em`,
+});
+
+const listStyles = css({
+  margin: 0,
+  padding: `${12 / perRem}em 0 ${6 / perRem}em 0`,
+  textIndent: 0,
```

Comment:
Haven't seen this before, usually `list-style: none` seems sufficient to reset everything? o.O